src/ReactImageMagnify.js

```diff
@@ -73,15 +73,17 @@ class ReactImageMagnify extends React.Component {
     }
 
     setSmallImageDimensionState() {
-        const {
-            offsetWidth: smallImageWidth,
-            offsetHeight: smallImageHeight
-        } = this.smallImageEl;
+        if (this.smallImageEl) {
```

Comment:
Thank you very much for opening this pull request! I really appreciate it. I opened a [pull request to your fork](https://github.com/smashercosmo/react-image-magnify/pull/1) with a different approach to the change at this line. My pull request makes the assumption that your change was to correct a failing test. If your change was motivated by something else please let me know.